Makefile

```diff
@@ -1,12 +1,13 @@
 GOPATH		:= $(shell go env GOPATH)
+GOPATH1		:= $(firstword $(subst :, ,$(GOPATH)))
```

Comment:
we need to pick the path which has our source code, not necessarily the first one.

Reply:
Actually what we care about here is where `go get` installs binaries, which (I believe) is always the first one.